bot/admin/test/xray/src/main/kotlin/XrayClient.kt

```diff
@@ -179,6 +183,11 @@ object XrayClient {
 
     fun saveStep(testKey: String, step: XrayBuildTestStep) = xray.saveStep(testKey, step).execute().body()
 
+    fun deleteStep(testKey: String, stepId: Int) =
```

Comment:
IMHO `Long` to `Int` conversion should be in `xray.deleteStep`call

Reply:
Moved